.github/workflows/release-prod.yml

```
@@ -0,0 +1,92 @@
+          node-version: 20
+          cache: 'pnpm'
+      - run: corepack enable
+      - name: Install deps (frontend)
+        working-directory: ./frontend
+        run: pnpm i --frozen-lockfile
+      - name: Build frontend
+        working-directory: ./frontend
+        run: pnpm build
+
+      - name: Deploy frontend dist
+        run: |
```

actionlint (1.7.7)

shellcheck reported issue in this script: SC2086:info:1:88: Double quote to prevent globbing and word splitting

(shellcheck)